Kernel/Heap/kmalloc.cpp

```diff
@@ -167,6 +167,29 @@ class KmallocSlabheap {
         return total;
     }
 
+    bool try_purge()
+    {
+        bool did_purge = false;
+
+        // FIXME: We cannot remove children from the list we are iterating through a structured loop,
+        //        so we have do iterate manually
```

Comment:
This can be a simple NOTE, nothing we can do to improve it.

Reply:
Also reworded it a bit